### PR TITLE
#159989012 Implement user signup functionality

### PR DESCRIPTION
**Pull request by Oloyedesinmiloluwa**
### What does this PR do?
Has user sign up implementation
### Description of Task to be completed?
- enable user registration
- display error details
- add unit and integration tests
### How should this be manually tested?
- `git clone the repo`
- `npm install`
- `npm run start` the start script
- Go to '/login' path and click on `register`
- Enter requested detail.
### What are the relevant pivotal tracker stories?
 [#159989014](https://www.pivotaltracker.com/story/show/159989012)
Screenshots (if appropriate)
image
image
image

### CODE REVIEW

File: src/redux/actions/signupAction.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+});
+export default userData => dispatch => axios.post(`${process.env.BACKEND_URL}/api/users`, userData)
+  .then((res) => {
+    const {
+      username, firstname, lastname, email, image,
+    } = res.data.user;
+    dispatch(setCurrentUser({
+      username, firstname, lastname, email, image,
+    }));
+    localStorage.setItem('token', res.data.user.token);
```
Comment (veeqtor) [inline src/redux/actions/signupAction.js]: Nice work sinmi, However, we already have a name for setting the token on local storage, I suggest you go with the agreed name for consistency.

File: src/styles/font.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+$description-font: "Abel", sans-serif;
```
Comment (veeqtor) [inline src/styles/font.scss]: This file is redundant, We already have a `fonts` file, You can go ahead and remove it.

File: src/styles/loginCard.scss [hunk 3]
```diff
@@ -1,9 +1,13 @@
+@import "./colours";
```
Comment (veeqtor) [inline src/styles/loginCard.scss]: These can also be removed since we are already importing them in the `index.scss`

**Review by veeqtor — APPROVED** [review]

**Review by Lumexralph — APPROVED** [review]

**Review by Lumexralph — CHANGES_REQUESTED** [review]
Please kindly check the format your PR message, the headers should be bolder than the body of the text.

**Review by Lumexralph — APPROVED** [review]

**Review by anuonifade — APPROVED** [review]
LGTM

**Pull request merged**